test/performance/README.md

```diff
@@ -1,128 +1,168 @@
-## Stress Test Experiments
+## KS Performance Experiments
 
 
 ## Generate the workload traffic
 
-1. Clone the following fork of the clusterloader2 repo: 
+1. Clone the following clusterloader2 repo: 
 
-```bash
-$ git clone -b ks-provider https://github.com/dumb0002/perf-tests.git
-```
+   ```bash
+   $ git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
 
-2. Configure the parameters of your experiment:  
+   Set the variable `WORK_DIR` to the path of the directory where you cloned your repo:
+
+   ```bash
+   $ export WORK_DIR=$HOME/perf-test/clusterloader2
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script:
+
+   For K8s clusters:
```

Comment:
Maybe say _plain_ Kubernetes, to distinguish more clearly from the OpenShift case.

Reply:
Fixed!